### PR TITLE
Removed some dances from the schedule

### PR DESCRIPTION
**Pull request by raphaelsattler**
And minors bugs fixed

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
-
+    <link rel="logo icon" href="/img/favicon.ico">
```
Comment (kaiomagalhaes) [inline _includes/head.html]: why did you remove this?

Comment (raphaelsattler) [reply _includes/head.html]: Actualy I added the line below on a space that was left over. Because of a bug in console looking for favicon that did not exist. So I created it.
